mitodl/ocw-studio
# update yarn configuration

#### What are the relevant tickets?

none

#### What's this PR do?

This upgrades us to Yarn 3, which is setup and operates fairly differently than before. Read about the differences [here](https://yarnpkg.com/getting-started/migration).

The biggest difference I actually opted out of (moving to the plug n' play dependency resolving scheme for node) because it didn't work out of the box and I didn't want to fiddle with it.

#### How should this be manually tested?

CI should be passing. If you pull down the branch the webpack build should work and whatnot.
